### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by vicmaburrito**
- [x] Create an npm project with webpack.
- [x] Write plain HTML markup with minimum styling (just to make the layout)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # Leaderboard
 The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+![screenshot](./photo.png)
+
```
Comment (elisha2kyakpo1) [inline README.md:6]: - [x] The image is not displaying, you may check the path and verify.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+## Built With
+
+- Javascript
+- Webpack
+- HTML 5
+- CSS 3
+
```
Comment (elisha2kyakpo1) [inline README.md:13]: - [x] The readme file is missing information on how a user like me can get this project and run it on my local machine.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hello @vicmaburrito,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @vicmaburrito, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**